gap/DirectSumDecomposition.gi

```diff
@@ -14,6 +14,10 @@ InstallMethod( DecomposeOnceByRandomEndomorphism,
     
     d := Maximum( List( ValuesOnAllObjects( F ), Dimension ) );
     
+	if d = 0 then
```

Comment:
Please use spaces instead of tabs. You can forbid your browser to use TABs as whitespaces.